code/go/0chain.net/smartcontract/dbs/event/authorizer_aggregate.go

```diff
@@ -0,0 +1,162 @@
+package event
+
+import (
+	"0chain.net/smartcontract/common"
+	"github.com/0chain/common/core/currency"
+	"github.com/0chain/common/core/logging"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+type AuthorizerAggregate struct {
+	gorm.Model
+
+	AuthorizerID string `json:"authorizer_id" gorm:"index:idx_authorizer_aggregate,unique"`
+	Round        int64  `json:"round" gorm:"index:idx_authorizer_aggregate,unique"`
+
+	Fee           currency.Coin `json:"fee"`
+	UnstakeTotal  currency.Coin `json:"unstake_total"`
+	TotalStake    currency.Coin `json:"total_stake"`
+	ServiceCharge float64       `json:"service_charge"`
+}
+
+func (edb *EventDb) ReplicateAuthorizerAggregate(p common.Pagination) ([]AuthorizerAggregate, error) {
+	var snapshots []AuthorizerAggregate
+
+	queryBuilder := edb.Store.Get().
+		Model(&AuthorizerAggregate{}).Offset(p.Offset).Limit(p.Limit)
+
+	queryBuilder.Order(clause.OrderByColumn{
+		Column: clause.Column{Name: "id"},
+		Desc:   false,
+	})
+
+	result := queryBuilder.Scan(&snapshots)
+	if result.Error != nil {
+		return nil, result.Error
+	}
+
+	return snapshots, nil
+}
+
+// nolint
+func (edb *EventDb) updateAuthorizerAggregate(round, pageAmount int64, gs *globalSnapshot) {
+	count, err := edb.GetAuthorizerCount()
+	if err != nil {
+		logging.Logger.Error("update_authorizer_aggregates", zap.Error(err))
+		return
+	}
+	size, currentPageNumber, subpageCount := paginate(round, pageAmount, count, edb.PageLimit())
+
+	exec := edb.Store.Get().Exec("CREATE TEMP TABLE IF NOT EXISTS temp_ids "+
+		"ON COMMIT DROP AS SELECT id as id FROM authorizers ORDER BY (id, creation_round) LIMIT ? OFFSET ?",
+		size, size*currentPageNumber)
+	if exec.Error != nil {
+		logging.Logger.Error("error creating temp table", zap.Error(exec.Error))
+		return
+	}
+
+	for i := 0; i < subpageCount; i++ {
+		edb.calculateAuthorizerAggregate(gs, round, edb.PageLimit(), int64(i)*edb.PageLimit())
+	}
+}
+
+// nolint
+func (edb *EventDb) calculateAuthorizerAggregate(gs *globalSnapshot, round, limit, offset int64) {
```

Comment:
add this method call to worker

Reply:
What exactly do you mean by above? I was seeing where `calculateBlobberAggregate` was added to worker for reference, but was unable to find one.

Reply:
strange, check here
https://github.com/0chain/0chain/blob/b5ff530fb9a3a7135b77d4b9b8f30ea761b2d3b3/code/go/0chain.net/smartcontract/dbs/event/blobber_aggregate.go#L72

Reply:
@dabasov 
Actually, I am using `edb. calculateAuthorizerAggregate` as [here].(https://github.com/0chain/0chain/blob/409c5bd43982f4f0e930bb6b3c4bcaf3d1e4fca3/code/go/0chain.net/smartcontract/dbs/event/authorizer_aggregate.go#L61)